src/executioners/MeshSolutionModify.C

```diff
@@ -80,7 +80,7 @@ MeshSolutionModify::endStep(Real input_time)
 }
 #endif
     }
-    _problem.computeUserObjects(EXEC_CUSTOM, UserObjectWarehouse::ALL);
+    _problem.computeUserObjects(EXEC_CUSTOM, Moose::AuxGroup::ALL);
```

Comment:
@dschwen Are you sure Hyrax is running with current MOOSE, this is wrong. It should be `Moose::ALL`.